features/emscripten/build.bat

```diff
@@ -0,0 +1,5 @@
+@echo off
+conan remote add bincrafters https://api.bintray.com/conan/bincrafters/public-conan
```

Comment:
Bincrafters remote is no longer needed, as `emsdk_installer` is already included in Conan Center
```suggestion
```